tools/RunScript.hx

```diff
@@ -127,19 +127,11 @@ class RunScript
 	{
 		var args = Sys.args();
 
-		var limeDirectory = Haxelib.getPath(new Haxelib("lime"), true);
-		var toolsDirectory = Path.combine(limeDirectory, "tools");
+		var cacheDirectory = Sys.getCwd();
 
-		if (!FileSystem.exists(toolsDirectory))
-		{
-			limeDirectory = Path.combine(limeDirectory, "..");
-			toolsDirectory = Path.combine(limeDirectory, "tools");
-		}
-
-		if (args.length > 2 && args[0] == "rebuild" && args[1] == "tools")
+		if (args.length > 0)
```

Comment:
This means the cwd is changed (not only `Haxelib.workingDirectory`) for all commands run, whereas previously it was only changed for `lime rebuild tools`. It might be better to keep the change more minimal and only change what's necessary which is `Haxelib.workingDirectory`.